lips/lip-10.md

```diff
@@ -0,0 +1,234 @@
+lip: 10
+title: Libra ID Spec
+authors: Andrey Chursin(@andll), Kevin Hurley (@kphfb)
+status: Draft
+type: Standard
+created: 11/03/2020
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+# Summary
+
+This LIP describes Libra ID - the human-readable identifier for user accounts.   
+
+In addition to describing the format of the Libra ID, this LIP also describes the off-chain protocol for the peer-2-peer payment with Libra ID.
+
+This document does not describe an off-chain protocol for using Libra ID for merchant use case - this protocol will be part of merchant flow LIP.  
+
+# Motivation
+Libra ID provides a convenient way to identify a user's account within a VASP. There are two main use cases for Libra ID:
+* For peer-2-peer payments, Libra ID allows a user to provide a human-readable ‘address’ to the sender. Basically, instead of sending a [lip-5 address](https://github.com/libra/lip/blob/master/lips/lip-5.md) (that needs to be refreshed for each payment, ideally), the receiver can just share their Libra ID once. In this case Libra ID plays the role of an email address for payments. Benefits for this use case:
+  * Libra ID is a user-friendly identifier, unlike lip-5 addresses
+  * The Libra ID resolution process can facilitate the exchange of rich information(e.g. nicknames, profile picture, etc.) about both the sender and receiver of the payment
+  * While it is encouraged for VASPs to treat subaddresses as single-use currently, Libra ID allows for this anonymity on-chain to be handled in a much simpler manner.  LibraID can be shared once with wallet apps transparently resolving the LibraID into a new identifier for each transaction (and thereby preventing linkability which could happen if subaddresses were re-used).
+* For the merchant use case, Libra ID allows users to self-identify to merchants. In this case, Libra ID plays the role of credit card number or also can be seen as a variant of OpenID-type protocol.
+
+## Examples in other blockchains
+Ripple proposes PayID. PayID has some similarity to the current proposal, but is not entirely identical.
+For example, PayID seems to target cross-chain payments, while libra id is intended to facilitate cross-vasp payments, but only inside the libra network.
+
+# Libra ID format
+Libra ID is a string in format `<user-id>$<libra-id-domain>`.
+
+* **user-id**, identifies an account within a VASP:
+  * Case insensitive
+  * Regex: `^[a-zA-Z0-9][a-zA-Z0-9-_+.]?$`
+  * Maximum length: 64 characters
+  * Not published on the chain, there is no way for an external entity to list all the users within VASP.
+* **libra-id-domain** identifies the VASP:
+  * Case insensitive
+  * Regex: `^[a-zA-Z0-9][a-zA-Z0-9-]?$`
+  * Maximum length: 63 characters
+  * libra-id-domains are published on the chain
+  * Each libra-id-domain is uniquely associated with single VASP
+  * VASPs can have multiple libra-id-domains associated, though it is not expected to be common
+
+## Multiple accounts per user
```

Comment:
So main motivation to this question is because some people might ask 'is libra id identifying a user, or specific account of the user". So this section is trying to clarify that. Do you think i need to add some clarification for it?